onlyoffice/.drone.star

```diff
@@ -0,0 +1,698 @@
+def main(ctx):
```

Comment:
This file is not needed. The extension has to be added to the top level `.drone.star` instead.